scripts/error-codes/codes.json

```diff
@@ -272,7 +272,7 @@
   "271": "Failed to replay rendering after an error. This is likely caused by a bug in React. Please file an issue with a reproducing case to help us find it.",
   "272": "The current renderer does not support hydration. This error is likely caused by a bug in React. Please file an issue.",
   "273": "Nesting of <View> within <Text> is not currently supported.",
-  "274": "Text strings must be rendered within a <Text> component.",
```

Comment:
Let's not touch this one. Instead a new one should be generated by running a script. (I think CI should tell its name?) Otherwise the message will be confusing for people running old versions and visiting the website.

Reply:
`yarn extract-errors`

Reply:
From looking at `scripts/error-codes/README`, looks like it's `yarn extract-errors`.

Thanks!

Reply:
Just noticed your second inline comment, lol. GitHub doesn't have realtime comments?!